pkg/sql/logictest/testdata/logic_test/views

```diff
@@ -1205,10 +1205,12 @@ DROP table t1nest CASCADE
 
 # Validate the objects being dropped
 query IT
-SELECT "reportingID", info::JSONB - 'Timestamp' - 'DescriptorID'
+SELECT "reportingID", info::JSONB - 'Timestamp' - 'DescriptorID' - 'CascadeDroppedViews'
 FROM system.eventlog
-WHERE "eventType" in ('drop_view', 'drop_table')
+WHERE "eventType" in ('drop_view', 'drop_table') AND
+(json_extract_path(info::JSONB, 'CascadeDroppedViews') @> '["db2.public.v3nest", "db2.public.v2nest", "db2.public.v1nest"]' OR
+ json_extract_path(info::JSONB, 'CascadeDroppedViews')  @> '["db2.public.v1nest", "db2.public.v2nest", "db2.public.v3nest"]')
```

Comment:
I don't understand this, both sides of this disjunction are the same. `a @> b` says that b's elements are a subset of a's without any order. 

Maybe:
```sql
(SELECT a <@ b AND a @> b
  FROM (
        SELECT json_extract_path(info::JSONB, 'CascadeDroppedViews') AS a,
               '
                ["db2.public.v3nest", "db2.public.v2nest", "db2.public.v1nest"]
               '::JSONB AS b
       ))
```